components/Table/Table.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div>
```

Comment:
no need for a div here, you can just put the table as the root component

Reply:
Done.